services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -95,7 +96,7 @@ async def upload_outputs(  # pylint:disable=too-many-statements  # noqa: PLR0915
     port_notifier: PortNotifier,
 ) -> None:
     # pylint: disable=too-many-branches
-    logger.debug("uploading data to simcore...")
+    _logger.debug("uploading data to simcore...")
```

Comment:
for this kind of logs, I guess you could use the log decorator and spare 1 line of code

Reply:
Not in this case. It's more complex. I will figure it out in the future, not in this PR, since it requires a bit of a refactor